CMakeLists.txt

```diff
@@ -96,6 +96,9 @@ if(BUILD_GDASBUNDLE)
   ecbuild_bundle( PROJECT mom6 GIT "https://github.com/jcsda-internal/MOM6.git" BRANCH main-ecbuild RECURSIVE )
   ecbuild_bundle( PROJECT soca GIT "https://github.com/jcsda-internal/soca.git" BRANCH develop )
 
+  # Build JEDI/DA or other peripherals
+  ecbuild_bundle( PROJECT gdas-utils SOURCE "./utils" )
```

Comment:
Curious:  where does project `gdas-utils` reside?

Reply:
it looks like it's just a separate CMake/ecbuild 'project' but inside of gdasapp (gdasapp/utils). We will have to do this for JEDI on WCOSS, as NCO does not allow ecbuild to 'clone'

Reply:
Got it.   I see the new `utils` directory.